integration-tests/src/test/java/oracle/kubernetes/operator/ItUsabilityOperatorHelmChart.java

```diff
@@ -292,10 +289,15 @@ public void testNotPreexistedOpServiceAccountCreateOperatorNegativeInstall() thr
   }
 
   /**
-   * Negative test : Helm will install the second operator with same target domain namespace as the
-   * first operator.
+   * Install operator usab-1 with taget DomainNameSpace [ usab-domainns-1 ]
+   * Install operator usab-2 with same target DomainNamesapce [ usab-domainns-1 ]
+   * Second operator MUST fail to intstall with following Exception  
+   * Error: rendered manifests contain a resource that already exists. 
+   * Unable to continue with install: existing resource conflict: 
+   * kind: RoleBinding, namespace: usab-domainns-1, 
+   * name: weblogic-operator-rolebinding-namespace
    *
-   * @throws Exception exception
+   * @throws Exception when second operator installation does not fail
```

Comment:
tests should not throw exceptions - tests should contain assertions that either pass or fail